src/pages/Home.jsx

```diff
@@ -1,7 +1,328 @@
+    },
+  ];
+
+  const serviceDetails = [
+    {
+      id: 1,
+      icon: <Scissors />,
+      title: 'Hair Cuts & Styling',
+    },
+    {
+      id: 2,
+      icon: <Star />,
+      title: 'Colors & Highlights',
+    },
+    {
+      id: 3,
+      icon: <UsersIcon />,
+      title: 'Treatment and Care',
+    },
+    {
+      id: 4,
+      icon: <ShoppingBag />,
+      title: 'Beauty Product',
+    },
+  ];
+
   return (
-    <div>
-      <h1>Home </h1>
+    <div className="bg-white container p-5 mx-auto px-4 md:px-8 lg:px-32">
+      <div className="flex flex-col  gap-5 pb-10">
+        {/* Welcome Images */}
+        <div className="w-full h-[350px] md:h-[500px] rounded-lg overflow-hidden flex justify-center items-center relative">
+          <div className="w-full h-full bg-gradient-to-r from-[#333333]/60 to-transparent z-30 absolute" />
+          {images.map((img, index) => (
+            <img
+              key={index}
+              src={img}
+              alt={`slide-${index}`}
+              className={`absolute w-full h-full object-cover transition-opacity duration-1000 ease-in-out ${
+                index === current ? 'opacity-100 z-20' : 'opacity-0 z-10'
+              }`}
+            />
+          ))}
+
+          {/* Welcome Message */}
+          <div className="flex flex-col justify-center items-center z-50 gap-5 md:gap-16">
+            <div className="bg-black/50 w-full md:w-[400px] rounded-lg flex flex-col items-center p-5">
+              <h1 className="text-white font-bold text-[20px] mb-5">
+                Style your perfect look
+              </h1>
+              <p className="text-white text-sm max-w-72 text-center">
+                Get personalized salon recommendations tailored to your beauty
+                needs.
+              </p>
+            </div>
+
+            {/* Search Inputs */}
+            <div className="bg-[#D9D9D9]/50 w-full md:w-[650px] flex flex-col md:flex-row justify-center items-center p-2 md:p-3 rounded-lg gap-4">
+              <SearchInputs
+                placeholder="Search for salons, services..."
+                Icon={Search}
+              />
+              <SearchInputs placeholder="Location" Icon={MapPin} />
+              <button className="bg-[#DB2777] p-2 md:p-3 w-full md:w-40 rounded hover:bg-[#be185d] cursor-pointer text-white font-bold">
+                Search
+              </button>
+            </div>
+          </div>
+        </div>
+
+        {/* How to use salon flow */}
+        <div className="mt-2 md:mt-5">
+          <h1 className="text-[#333333] text-2xl text-center font-bold mb-2 md:mb-3">
+            How Salon-Flow Works
+          </h1>
+          <p className="text-[#333333] font-thin text-center">
+            Connecting clients with salons has never been easier
+          </p>
+          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-5 md:gap-12 mt-5 md:mt-10">
+            {HowDetails.map((how) => (
```

Comment:
To make the code clear and easy to understand, when you use `.map()` (or similar array methods), the parameter name in the callback should be the singular form of the array name.  

For example, if the array is named `students`, the callback parameter should be named `student`:  

```javascript
const students = ['Alice', 'Bob', 'Charlie'];

students.map((student) => {
  return student.toUpperCase();
});
```

This makes it obvious that you are working with one item from the students array. Using clear and consistent naming helps make the code easier to read and reason about.